weaver/processes/wps_package.py

```diff
@@ -1962,9 +1979,34 @@ def _handler(self, request, response):
                 # ADES execute the cwl locally
                 loading_context = None
 
-            wps_out_dir_prefix = os.path.join(get_wps_output_dir(settings), "tmp")
-            runtime_args = {"no_read_only": True, "outdir": self.workdir, "tmp_outdir_prefix": wps_out_dir_prefix}
-            runtime_context = RuntimeContext(kwargs=runtime_args)
+            self.update_effective_user()
+            self.update_requirements()
+
+            # note:
+            #   Parameter 'weaver.wps_workdir' is the base-dir where sub-dir per application packages will be generated.
+            #   Parameter 'self.workdir' is the actual location PyWPS reserved for this process (already with sub-dir).
+            #   If no 'weaver.workdir' was provided, reuse PyWps parent workdir since we got access to it.
```

Comment:
Since it's already a little bit confusing, we should make sure that there is no error in doc. I guess here we should read weaver.wps_workdir and not weaver.workdir?

Reply:
Indeed. Fixing it.